cmd/main.go

```diff
@@ -29,6 +29,7 @@ func main() {
 	flag.BoolVar(&ctrlOpts.EnablePprof, "dangerous-enable-pprof", false, "If set to true, enable pprof on "+controller.PprofListenAddr)
 	flag.DurationVar(&ctrlOpts.APIRequestTimeout, "api-request-timeout", time.Duration(0), "HTTP timeout for Kubernetes API requests")
 	flag.BoolVar(&runController, controllerinit.InternalControllerFlag, false, "[Internal] run the controller code")
+	flag.BoolVar(&ctrlOpts.APIPriorityAndFairness, "enable-APIPriorityAndFairness", true, "gate for using flowcontrol beta APIs, must be false for clusters <= v1.19")
```

Comment:
Maybe `Enable/disable APIPriorityAndFairness feature gate for apiserver. Recommended to disable for <= k8s 1.19.`